src/firefly/js/util/Validate.js

```diff
@@ -100,6 +100,32 @@ export const validateUrl = function(description,valStr) {
     return retval;
 };
 
+export const validateDate = function(description, valStr){
+    var retval = {
+        valid: true,
```

Comment:
use `const` and `let` instead of `var`. This file predates our ES6 conversion but any new code should use `let` instead of `var`.

Reply:
Done!